Assets/Scripts/CollectObjects.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CollectObjects : MonoBehaviour
+{
+    private ThreeDimensionalCalculate aCalculate = new ThreeDimensionalCalculate();
+    private CurrentGameObjects GameObjects = CurrentGameObjects.Instance;
+
+    void Update()
+    {
+        // Finds the player
+        GameObject Player = GameObject.FindGameObjectsWithTag("Player")[0];
```

Comment:
this is fine but there is also a FindGameObjectWithTag that just returns a single object for future use